packages/gestalt/src/useResponsiveMinWidth.js

```diff
@@ -0,0 +1,66 @@
+// @flow strict
+import { useCallback, useState, useMemo, useEffect } from 'react';
+import {
+  addListener,
+  removeListener,
+  supportsMatchMedia,
+} from './utils/matchMedia.js';
+
+type MinWidthType = 'lg' | 'md' | 'sm' | 'xs';
+
+export default function useResponsiveMinWidth(): ?MinWidthType {
+  const breakpoints = useMemo(
```

Comment:
Seems like there's no need to `useMemo` these instead of defining outside the hook. That would also let you not `useCallback` getMinWidth as well

Reply:
DONE